src/ci/scripts/install-mingw.sh

```diff
@@ -27,8 +27,8 @@ IFS=$'\n\t'
 
 source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
 
-MINGW_ARCHIVE_32="i686-6.3.0-release-posix-dwarf-rt_v5-rev2.7z"
```

Comment:
I'll leave a note here that the comment at the top of this file will need to be updated should this land.